fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/mixin/item/group/client/MixinCreativePlayerInventoryGui.java

```diff
@@ -120,9 +120,9 @@ private void init(CallbackInfo info) {
 		fabric_updateSelection();
 
 		int xpos = x + 170;
```

Comment:
I see no code here to change the button positions, I forked this and what I got is this:
![image](https://user-images.githubusercontent.com/34910653/80822916-3da6e500-8c0e-11ea-89b2-cbfb8cb2cd75.png)

Reply:
Huh. Probably lost that change during the rebase. I'll fix it and push soon.

Reply:
Please do.